src/aws/user.tf

```diff
@@ -149,6 +149,7 @@ resource "aws_iam_user_policy" "live-streaming-policies" {
         "mediapackage:UpdateOriginEndpoint",
         "mediapackage:TagResource",
         "mediapackage:CreateHarvestJob"
+        "mediapackage:ListHarvestJobs"
```

Comment:
Did you try to apply this change with terraform ? You will have a surprise...

Reply:
Oops, sorry